<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acknext MIEP</string>
  </property>
  <widget class="QAcknextWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScene_View"/>
   <addaction name="actionLight_View"/>
   <addaction name="actionNormal_View"/>
   <addaction name="actionWireframe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>&amp;Model Structure</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Model</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Root</string>
        </property>
        <item>
         <property name="text">
          <string>Node A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Node B</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Meshes</string>
        </property>
        <item>
         <property name="text">
          <string>Mesh A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mesh B</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Materials</string>
        </property>
        <item>
         <property name="text">
          <string>Material A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Material B</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Mate&amp;rial Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="fileToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource/ackui.qrc">
     <normaloff>:/icons/folder-open.png</normaloff>:/icons/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/ackui.qrc">
     <normaloff>:/icons/content-save.png</normaloff>:/icons/content-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../resource/ackui.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>O&amp;ptions</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNormal_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionLight_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionScene_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scene View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAcknextWidget</class>
   <extends>QWidget</extends>
   <header location="global">qacknextwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/ackui.qrc"/>
 </resources>
 <connections/>
</ui>
